packages/apps/spaces/pages/api/.ory/[...paths].ts

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2022 Ory Corp
```

Comment:
this file is duplicated

Reply:
https://github.com/openline-ai/openline-customer-os/issues/874